app/models/facility_group.rb

```diff
@@ -138,6 +138,10 @@ def cohort_analytics(period:, prev_periods:)
     query.call
   end
 
+  def syncable_patients
+    registered_patients.with_discarded
```

Comment:
🪓 Don't think this is called, right?  Everything will go through Region or Patient I believe.

Reply:
This is called [by `Patient` like so](https://github.com/simpledotorg/simple-server/pull/1333/files/59aab26c1b0ac4ad3791b388c0c2af76b21da996#diff-94dcc014f62f0d9b6084ad98f82f74cd40c86b029ef1691914bdbc960c2f42dcR59). This method is how we keep sync backwards compatible.